docs/source/design/composite-time-series.rst

```diff
@@ -0,0 +1,220 @@
+#####################
+Composite Time Series
+#####################
+
+Purpose
+=======
+
+It is a challenge for users to identity what the correct authoritative time series is for a given measurement at a location. Additionally these time series often change over time, either being completely new or changing their interval as newer technologies become available.
+
+Gather an entire Period of Record for the value at a location is also rather difficult. And the POR record and "authoritative timeseries" may be one-in-the same.
+
+
+Need
+====
+
+#. CWMS and Access-2-Water require a simple mechanism to allow users of data to retrieve the Authoritative Period of Record data for a given measurement without having to  understand all of the possible component time series that may be involved. 
+#. Period-of-Record time series *should* not be created by duplicating data from the component time series and merging them into a new one.
+#. The naming of the time series should fit within the excepting CWMS Time Series Identifier design and not unreasonably interfere with existing usages.
+
+
+Caveats
+=======
+
+#. It is assumed that CWMS-Vue will, as-always, require updates to handle what is created here.
+   #. e.g. we're not going to let any current limitations of CWMS-Vue hinder our design.
+
+
+Proposal
+========
+
+Description
+-----------
+
+CDA should handle a concept of a "Composite Time Series". Whether a Time Series is considered composite will be determined by a specific element of the Time Series Identifier.
+Data Administrators will configure which Time Series, and the range there-in, are part of the composite time series.
+CDA will use this stored information to build the Time Series per the question.
+
+Additional names not used
+-------------------------
+
+#. Virtual Time Series
+#. Period of Record Time Series
+
+Both names have been discarded. We use "Virtual" in too many other places with a more direct meaning of that word. 
+For Period-of-Record, while that is the primary use-case, the concept is useful in other situations as well.
+
+Hence generically have have a "composite time series"
+
+Axioms
+------
+
+#. Composite Time Series are Irregular
+#. The definition of the composite time series is stored within the CWMS database
+#. The members of a composite time series define a continuous range
+   #. The date ranges of a composite time series *MUST* not overlap
```

Comment:
> The members of a composite time series define a continuous range
This could be worked around by using versioning in the underlying implementation. If there's ambiguity (which I 100% guarantee will happen), then pick whichever is the most recent version.
Alternatively, for a simpler implementation, it could use the "archived" flag on the timeseries. If a timeseries is archived, then we can decide on a behavior based on the start-end dates of that timeseries.